ui/js/jquery.pods.js

```diff
@@ -1852,6 +1852,12 @@
     if ( typeof pods_localized_strings == 'undefined' ) {
         pods_localized_strings = {
```

Comment:
Format:

- Prefix with __ just like __() functions
- Convert original string to all lowercase
- Remove punctuation and HTML tags from original string
- Replace spaces with an underscore _